app/src/main/java/com/keylesspalace/tusky/util/NotificationHelper.java

```diff
@@ -268,7 +268,7 @@ public static void make(final Context context, Notification body, AccountEntity
         summaryStackBuilder.addParentStack(MainActivity.class);
         summaryStackBuilder.addNextIntent(summaryResultIntent);
 
-        PendingIntent summaryResultPendingIntent = summaryStackBuilder.getPendingIntent(notificationId,
+        PendingIntent summaryResultPendingIntent = summaryStackBuilder.getPendingIntent((int) (notificationId + account.getId() * 10000),
```

Comment:
Looks like these ids need to be unique or the behavior of previously created Intents is changed 🙄